templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>Let's default this as an argument to non-zero mean intensity, and remove the description of what to change. Power users can change if they want</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>